File: corpus/2017q1/_p2q3_ok/_check/F;0645.java

import java.util.Scanner;

public class Questao3P2 {

    public static void main(String[] args) {

        int l=0, c=0;
        int i, k;
        int[][] A=new int [l][c];
        Scanner leitor= new Scanner (System.in);
        System.out.println("Digite um numero par de linhas:");
        l = leitor.nextInt();
         System.out.println("Digite um numero par de colunas:");
        c = leitor.nextInt();
        if (l%2!=0 || c%2!=0 ){
            System.out.println("Apenas numeros pares, rode o programa novamente!");
           }
        for (i=0;i<l;i++){
            for(k=0;k<c;k++){

                System.out.println("Digite o valor da linha "+i+" coluna "+k);
                A[i][k]=leitor.nextInt();
            }

        }

    }

}

File: corpus/py/F;0645.py
int l=0, c=0
int i, k
Scanner leitor= new Scanner (System.in)
print("Digite um numero par de linhas:")
l = int(input())
print("Digite um numero par de colunas:")
c = int(input())
if l%2!=0 || c%2!=0 :
    print("Apenas numeros pares, rode o programa novamente!")
for i=0 in range(0, l):
    for k=0 in range(0, c):
        print("Digite o valor da linha "+i +" coluna "+k)